migrate/migrate.go

```diff
@@ -100,6 +112,54 @@ func (m *Migrator) migrateToV0_1_0(id uuid.UUID) error {
 	return nil
 }
 
+// migrateToV0_12_0 migrates a project to the version v0.12.0
+//
+// Steps:
+// - 1. Reset project state recreate initial accounts
+// - 2. Get all accounts for project and update with shifted addresses and removed unused fields
+func (m *Migrator) migrateToV0_12_0(projectID uuid.UUID) error {
+	// 1. reset project state
+	createdAccounts, err := m.projects.Reset(&model.InternalProject{
+		ID: projectID,
+	})
+	if err != nil {
+		return errors.Wrap(err, "migration failed to reset project state")
+	}
+
+	var accounts []*model.InternalAccount
+	err = m.store.GetAccountsForProject(projectID, &accounts)
+	if err != nil {
+		return errors.Wrap(err, "migration failed to get accounts")
+	}
+
+	if len(accounts) != len(createdAccounts) {
+		return fmt.Errorf("migration failture, created accounts length doesn't match existing accounts")
+	}
+
+	// 2. migrate accounts
+	for i, acc := range accounts {
+		acc.Address = createdAccounts[i].Address
+		acc.DraftCode = adapter.ContentAddressFromAPI(acc.DraftCode)
+
+		err = m.store.DeleteAccount(acc.ProjectChildID)
```

Comment:
we delete and then create the same account so we get rid of old values we no longer need. We could just update but for simplicity to not extend the store interface we used these two methods, it's not used a lot so it shouldn't matter